config/site_settings.yml

```
@@ -2423,6 +2423,11 @@ uncategorized:
     default: true
     hidden: true
 
+  splash_screen:
+    default: false
+    client: true
+    hidden: true
```

I don't think we're actually using this on the client, so

```suggestion
  splash_screen:
    default: false
    hidden: true
```